### ISSUE TITLE
A dependency job for rpcbind.service failed

### ISSUE THREAD

**Issue by billb2050** [obsolescence/pidp11#4]
**Install PiDP-11 package into OS? yes**
Synchronizing state of rpcbind.service with SysV service script with /lib/systemd/systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install enable rpcbind
Created symlink /etc/systemd/system/multi-user.target.wants/rpcbind.service → /lib/systemd/system/rpcbind.service.
Created symlink /etc/systemd/system/sockets.target.wants/rpcbind.socket → /lib/systemd/system/rpcbind.socket.
**A dependency job for rpcbind.service failed. See 'journalctl -xe' for details.**

Autostart the PDP-11 using the GUI(Y) or .profile (H)?
-- Y recommended, H is for headless Pis without GUI:^C

pi@pidp-1170-p4:/opt $ **journalctl -xe**
░░ Support: https://www.debian.org/support
░░ 
░░ A start job for unit rpcbind.socket has finished with a failure.
░░ 
░░ The job identifier is 1545 and the job result is failed.
Dec 18 22:12:00 pidp-1170-p4 systemd[1]: Dependency failed for rpcbind.service >
░░ Subject: A start job for unit rpcbind.service has failed
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ A start job for unit rpcbind.service has finished with a failure.
░░ 
░░ The job identifier is 1541 and the job result is dependency.
Dec 18 22:12:00 pidp-1170-p4 systemd[1]: rpcbind.service: Job rpcbind.service/s>
Dec 18 22:12:00 pidp-1170-p4 systemd[1]: Reached target rpcbind.target - RPC Po>
░░ Subject: A start job for unit rpcbind.target has finished successfully
░░ Defined-By: systemd
░░ Support: https://www.debian.org/support
░░ 
░░ A start job for unit rpcbind.target has finished successfully.
░░ 
░░ The job identifier is 1546.
Dec 18 22:12:00 pidp-1170-p4 sudo[2458]: pam_unix(sudo:session): session closed>
lines 2791-2813/2813 (END)

**Comment by obsolescence** [comment]
Bill, apologies for being far too slow with responding. What OS were you running? I only support the latest Bookworm, might that be the explanation?

**Comment by Terri-Kennedy** [comment]
I agree with Oscar - Raspberry Pi OS Bookworm (or shortly the new Trixie in my Terri-Kennedy/pidp11 fork) is the minimum release needed to run the emulator, either from the code here or in my fork. If @billb2050 is still following this issue, please confirm you are using Raspberry Pi OS Bookworm.

Also, note that the install script will do its best to install the PiDP-11 software on amd64 (PC, as opposed to Pi) Debian-based Linux releases (again, Bookworm or later). But there are quite a few Debian-based distros around, and for that reason installing on a non-Pi system will display a "for experts only" caution in my fork.
